BootloaderCorePkg/Stage1B/Stage1B.c

```diff
@@ -125,6 +125,35 @@ PrepareStage2 (
   return Dst;
 }
 
+/**
+  Check digest for external hash store component.
+
+  @param[in] Stage1bParam   Stage1B parameters.
+  @param[in] Src            Extrenal hash store address
+  @param[in] Length         Used length of external hash store
+
+**/
+VOID
+CheckExtHashStoreMbDigestExtend (

```

Comment:
This method is updating the hash of the KeyManifestHash store if the hash type does NOT matches with current PCR enabled bank. I would prefer to call this method as : UpdateKeyManifestHashForTPMExtend()

Reply:
Thanks. Will do.